### ISSUE TITLE
wf-shell-0.3.1: wf-background: symbol lookup error

### ISSUE THREAD

**Issue by mamins1376** [void-linux/void-packages#20618]
I'm using Wayfire as display server. Trying to utilize any executable provided by `wf-shell` results in symbol lookup error. Maybe a linking error?

* xuname: Void 5.4.28_1 x86_64 AuthenticAMD notuptodate rrrmFFFFFFF
* package:  wf-shell-0.3.1

(_notuptodate_ because I use a tier 2 mirror.)

### Expected behavior
it should Just Work(TM).

### Actual behavior
```
$ wf-background
wf-background: symbol lookup error: wf-background: undefined symbol: _ZN12wf_animation6circleE
```

Same also happens with both `wf-panel` and `wf-dock`.

### Steps to reproduce the behavior

1. `# xbps-install wf-shell`
2. Use any binary provided by the package to see the results.

**Comment by xtraeme** [comment]
And this is the affected symbol: https://github.com/WayfireWM/wf-config/commit/211a86c35419871ca8d47c3d50d128a326e9d084#diff-dec55d1344f62aea304c7e727402c2e8L7

**Comment by mamins1376** [comment]
Would a simple patch version bump fix trigger that? Maybe landing the fix by upstream repo takes place after an actual release

**Comment by xtraeme** [comment]
try to rebuild it with xbps-src, but I'm not sure if current wf-shell will be API compatible against wf-config-0.4.0. Otherwise apply the wf-shell-0.4 PR and build it.

**Comment by xtraeme** [comment]
our buildbot builder has died due to qt5!

**Comment by xtraeme** [comment]
@mamins1376 you should open the issue in wf-config not wf-shell :-)

**Comment by mamins1376** [comment]
@xtraeme yea realized it a little bit late :) I'm confused, now `wf-config` should be bumped to 0.4.0._3 right?

**Comment by xtraeme** [comment]
No, apply the wf-shell-0.4 PR and build it locally.

**Comment by xtraeme** [comment]
fixed by #20592
